bin/alluxio-process

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env bash
```

Comment:
`bin/alluxio-process.sh` to be consistent with other util scripts under bin/?

Reply:
I do my best when writing shell to follow google style when writing these script, and the guide is ambivalent about extensions

https://google.github.io/styleguide/shell.xml#File_Extensions

Additionally, the decision to leave the extension off of the script was intentional. The script results in the PID of the shell process being replaced with actual PID of the Alluxio process I felt it was appropriate to leave the extension off as this is script is more like a "real executable" than a script which kicks off some daemon processes (like `alluxio-start.sh`).


I also am curious about your opinion on the name. I just named it `alluxio-process` to start. But maybe `launch-process` or `alluxio-launch` could also work?

Reply:
SG.
`launch-process` or `process-launcher`?

Reply:
renamed to `launch-process`